common/types/data_connections.ts

```diff
@@ -13,13 +13,33 @@ export interface PermissionsConfigurationProps {
   hasSecurityAccess: boolean;
 }
 
+export interface TableColumn {
+  name: string;
+  dataType: string;
+}
+
+export interface Acceleration {
+  name: string;
+  status: 'ACTIVE' | 'INACTIVE';
```

Comment:
can you please create a type for status so we can re-use it at other places

Reply:
Nice catch. done.